examples/pipelines/fl_cross_silo_factory/submit.py

```diff
@@ -0,0 +1,308 @@
+"""Federated Learning Cross-Silo pipeline built by a factory class.
```

Comment:
It would be good to have very clear guidance (either at top of this demo submit.py file, or in a separate readme who also explained how to use fl_factory to write/submit fl experiment.) on where users need to touch/modify in such submission script when adapting it to their own FL scenario. E.g., based on my reading, roughly speaking, seems users might need to touch config.yaml+ section C (FL steps) + section D (adjust accordingly to their FL config)?

Reply:
Absolutely, let me write some more guidance in the beginning of the file. We will need a proper tutorial on this but I'd like to take more time to write it up in a distinct PR.

Reply:
Filed as #80